src/components/views/sul-study-place/filtering-table/study-place-filtering-table.tsx

```diff
@@ -108,9 +108,9 @@ const StudyPlaceFilteringTable = ({items}: Props) => {
       <form onSubmit={e => e.preventDefault()} action="javascript:void(0);">
         <fieldset className="text-18">
           <legend className="sr-only">Filter places to study</legend>
-          <div className="mb-30 flex w-full flex-wrap items-center justify-around gap-15 *:min-w-300 *:flex-1 *:3xl:min-w-0">
+          <div className="mb-30 flex w-full flex-col flex-wrap items-center justify-center gap-15 *:w-full sm:flex-row sm:*:w-fit">
```

Comment:
Do we need to do this on the other filters displays?

Reply:
Darcy request this change specifically for the "Places to Study" page 😃

Reply:
Ok :+1: